First update of KEB docs (#35)
kyma-project/kyma-environment-broker: docs/08-02-deprovisioning-kyma-environment.md

> # Deprovision SAP BTP, Kyma runtime using KEB

Reviewer: ```suggestion
# Deprovision SAP BTP, Kyma runtime using Kyma Environment Broker
```

> This tutorial shows how to deprovision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker.

Reviewer: ```suggestion
This tutorial shows how to deprovision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker (KEB).
```

   export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
   ```

> 3. Make a call to the Kyma Environment Broker to delete a Kyma runtime on Azure.

Reviewer: ```suggestion
3. Make a call to KEB to delete a Kyma runtime on Azure.
```
